patterns/rightsizing-storage-volumes.md

```diff
@@ -0,0 +1,43 @@
+# Rightsizing storage volume
+
+## Version
+1.0
+
+## Submitted By
+Sarah Hsu
+
+## Published Date
+TBD
+
+## Intent
+Choose resource storage that is the right size for the task at hand.
+
+## Tags
+Cloud, Storage, Cloud Engineer, Small
+
+## Problem
+From both an embodied carbon angle and an energy proportionality angle, it's better to delete idle storage resources or maximise storage utilisation, so the storage layer is optimised for the task. 
+
+From the embodied carbon angle, the unused capacity of underutilised storage could be more efficiently used by another task or process. From the energy proportionality angle, two storage running at low utilisation rates will consume more energy than one storage running at a high utilisation rate.
+
+## Solution
+Reduce idle storage resources by 
+1. Delete any unused storage resource
+2. Set a retention policy on storage resources to automate the deletion of unused storage resources 
+3. Move storage resources away from underutilised storage
```

Comment:
Would be good to add two patterns here, one for managed storage resources such as S3 one for unmanaged storage resources such as drives and long term storage solutions.

Reply:
We could, but they both would have v similar solution. How about we use tags to encapsulate that the pattern can be for both managed and unmanaged storage?